esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -4028,11 +4041,18 @@ void CWsWuFileHelper::readWUFile(const char* wuid, const char* workingFolder, Ws
     case CWUFileType_CPP:
     case CWUFileType_LOG:
     {
-        const char *tail=pathTail(item.getName());
-        fileName.set(tail ? tail : item.getName());
+        const char *file=item.getName();
+#ifndef _CONTAINERIZED
+        validateFilePath(file, false, "run", nullptr, nullptr);
+#else
+        validateFilePath(file, false, "query", nullptr, nullptr);
```

Comment:
This is potentially not going to work for containerized systems - you can have c++ files and dlls stored in multiple locations.  An unusual case though.
The idea that you can get a single location and then compare it as a prefix is likely to have other instances (like log overrides in bare metal) where it doesn't really work.

Reply:
Any suggestion about how to do with the c++ files and dlls stored in multiple locations?